user-management/src/main/elm/sources/DataTypes.elm

```diff
@@ -83,10 +82,9 @@ type Msg
     | Login String
     | AddRole User String
     | RemoveRole User String
-    | SubmitUpdatedInfos User
-    | SubmitNewUser User Password
+    | SubmitUpdatedInfos User (List String)
```

Comment:
I think the list of role you want to save must be a part of the user you are updating